lib/index.js

```diff
@@ -209,7 +209,7 @@ exports = module.exports = internals.Podium = class {
                 }
 
                 const update = internals.flag('clone', handler, event) ? Hoek.clone(data) : data;
-                const args = internals.flag('spread', handler, event) && Array.isArray(update) ? update : [update];
+                const args = internals.flag('spread', handler, event) && Array.isArray(update) ? update.slice(0) : [update];
```

Comment:
This can be simplified to:

```js
const args = internals.flag('spread', handler, event) ? [].concat(update) : [update];
```

Note that it has the subtle side-effect of respecting [`@@isConcatSpreadable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/isConcatSpreadable), which I expect is fine / more appropriate.

Reply:
`concat` seems to be 25% slower than `slice` for this kind of operation, is it worth it?

Reply:
Nah, just keep it then.

Reply:
Btw. my testing shows it to be 84% slower for arrays, and 94% for objects / values.